sdk-estimate-gas/README.md

```diff
@@ -34,125 +34,88 @@ This calculation is complicated by the fact that the major cost is the cost of w
 
    - `OPTIMISM_GOERLI_URL` is the URL for Optimism Goerli, if you use [a different node provider](https://community.optimism.io/docs/useful-tools/providers/).
 
+   - `OPTIMISM_MAINNET_URL` is the URL for Optimism Mainnet, if you use [a different node provider](https://community.optimism.io/docs/useful-tools/providers/).
+
 
 1. Use Node to run the script
 
    ```sh
-   node gas.js
+   node gas.js --network mainnet
    ```
 
-That's it.
+   The command line options are:
 
-### Results
+   - `--network`: The network to estimate gas on:
+     - `mainnet`: The Optimism mainnet network
+     - `goerli`: The Optimism testnet on Goerli
+     - `bedrock-alpha`: The Bedrock alpha test network
 
-The results you get from the script on Optimism Goerli are different than the ones you'd get from running on the main Optimism blockchain. 
-The reason is that on Goerli L1 gas costs is just seven times the L1 gas cost.
-On the main network the ratio is usually in the tens of thousands ([click here to see the current values](https://public-grafana.optimism.io/d/9hkhMxn7z/public-dashboard?orgId=1&refresh=5m)).
+   - `--verify`: Run the transaction to verify the estimate
+  
+     Note that right now there is a bug that prevents the combination `--network bedrock-alpha --verify` from working.
 
-Here is an example of results from the main Optimism blockchain.
+### Results
+
+Here is an example of results from the main Optimism blockchain:
 
 
 ```
-ori@Oris-MBP sdk-estimate-gas % ./gas.js 
+ori@Oris-MBP sdk-estimate-gas % ./gas.js --network mainnet --verify
+ori@Oris-MacBook-Pro sdk-estimate-gas % ./gas.js --network mainnet --verify
 About to get estimates
 About to create the transaction
-Transaction created, submitting it
+Transaction created and submitted
 Transaction processed
 Estimates:
-   Total gas cost:       99958954362216 wei
-      L1 gas cost:       99926386362216 wei
+   Total gas cost:       58819800030256 wei
+      L1 gas cost:       58787232030256 wei
       L2 gas cost:          32568000000 wei
 
 Real values:
-   Total gas cost:       99493733721792 wei
-      L1 gas cost:       99461179721796 wei
-      L2 gas cost:          32553999996 wei
+   Total gas cost:       58819786030272 wei
+      L1 gas cost:       58787232030256 wei
+      L2 gas cost:          32554000016 wei
 
 L1 Gas:
-      Estimate:       4296
+      Estimate:       4276
           Real:       4276
-    Difference:        -20
+    Difference:          0
 
 L2 Gas:
       Estimate:      32568
           Real:      32554
     Difference:        -14
-
 ```
 
-The L1 gas cost is thousands of times the L2 gas cost.
+The L1 gas cost is over a thousand times the L2 gas cost.
 This is typical in Optimistic transactions, because of the cost ratio between L1 gas and L2 gas.
 
 
 
 ## How does it work?
 
-In this section we go over the script line by line to learn how to use the SDK for gas estimates.
```

Comment:
Why did we nuke this again?

Reply:
Mostly because I didn't think it was worth keeping and updating the explanation for the parts of the JavaScript that are just standard and not related to our SDK. Would you rather I added them back up?

Reply:
hm. Does it help people understand what is happening/make the docs more noob friendly? Or is it more extra info

Reply:
Depends on the type of noobness. A crypto-noob won't need the extra info (and might even find it annoying to slog through). A JavaScript noob would need it.

So I'm going to cheat and hide these explanations in a "details" section.

Reply:
Done. Ready for rereview.